opus/src/components/signin/SignIn.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import logo from '../singnup/logo-OPUS.png';
+import './SignIn.css';
+
+export default class SignIn extends Component {
+  state = {
+    email: '',
+    password: ''
+  };
+
+  loginUser = () => {
+    axios
+      .post('http://localhost:8000/users/signin', {
+        email: this.state.email,
+        password: this.state.password
+      })
+      .then(function(res) {
+        console.log(res);
+      })
+      .catch(function(error) {
+        console.log(error);
+      });
+  };
+
+  handleChange = event => {
+    const { value, name } = event.target;
+    this.setState({ [name]: value });
+  };
+
+  handleSubmit = event => {
+    this.setState({
```

Comment:
Are you sure you want to clean state in all situations ?

Reply:
yes